### PR TITLE
[FEATURE] Implement the "column.standard_deviation" metric for sqlite database

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- JIRA: GREAT-586
-
-

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: great_expectations/expectations/metrics/column_aggregate_metrics/column_mean.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def _pandas(cls, column, **kwargs):
     def _sqlalchemy(cls, column, **kwargs):
         """SqlAlchemy Mean Implementation"""
         # column * 1.0 needed for correct calculation of avg in MSSQL
-        return sa.func.avg(column * 1.0)
+        return sa.func.avg(1.0 * column)
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/metrics/column_aggregate_metrics/column_mean.py:28]: Just curious: does this make a difference in the results of the calculation?

Comment (alexsherstinsky) [reply great_expectations/expectations/metrics/column_aggregate_metrics/column_mean.py:28]: @Shinnnyshinshin No -- only sense of aesthetics on my part.  Thanks!

File: great_expectations/expectations/metrics/column_aggregate_metrics/column_standard_deviation.py [hunk 2]
```diff
@@ -34,15 +38,56 @@ def _pandas(cls, column, **kwargs):
...
         if _dialect.name.lower() == "mssql":
             standard_deviation = sa.func.stdev(column)
+        elif _dialect.name.lower() == "sqlite":
+            mean = _metrics["column.mean"]
+            nonnull_row_count = _metrics["column_values.null.unexpected_count"]
+            standard_deviation = sa.func.sqrt(
+                sa.func.sum((1.0 * column - mean) * (1.0 * column - mean))
+                / ((1.0 * nonnull_row_count) - 1.0)
+            )
         else:
```
Comment (Shinnnyshinshin) [inline great_expectations/expectations/metrics/column_aggregate_metrics/column_standard_deviation.py:52]: ❤️

File: great_expectations/self_check/util.py [hunk 3]
```diff
@@ -1496,7 +1496,6 @@ def candidate_test_is_on_temporary_notimplemented_list_cfe(context, expectation_
             "expect_column_values_to_be_dateutil_parseable",
             "expect_column_values_to_be_json_parseable",
             "expect_column_values_to_match_json_schema",
-            "expect_column_stdev_to_be_between",
```
Comment (Shinnnyshinshin) [inline great_expectations/self_check/util.py:1499]: Whoohoo. Just explicitly stating that this enables the `expect_column_stdev_to_be_between` Expectation tests for sql dialects (include `sqlite`). So additional unittests are not needed for this PR

**Review by Shinnnyshinshin — APPROVED** [review]
Really appreciate you digging into this @alexsherstinsky Thank you very very much. LGTM

**Review by kenwade4 — APPROVED** [review]
Looks great and everything is passing for `expect_column_stdev_to_be_between` in the build_gallery.py script! Thanks!

**Pull request merged**